homeassistant/components/hue/v2/binary_sensor.py

```diff
@@ -172,39 +206,26 @@ def __init__(
 
 
 # pylint: disable-next=hass-enforce-class-module
-class HueMotionAwareSensor(HueMotionSensor):
-    """Representation of a Motion sensor based on Hue Motion Aware.
+class HueMotionAreaConfigurationSensor(HueAreaMotionSensor):
+    """Base class for motion sensors linked to a motion area configuration."""
 
-    Note that we only create sensors for the SecurityAreaMotion resource
-    and not for the ConvenienceAreaMotion resource, because the latter
-    does not have a state when it's not directly controlling lights.
-    The SecurityAreaMotion resource is always available with a state, allowing
-    Home Assistant users to actually use it as a motion sensor in their HA automations.
-    """
```

Comment:
See this note I made in the code. ConvenienceArea sensors are ONLY available when you use group already to control lights, where the security ones are always available. 
With this Pr users end up with 2 sensors for the same, of which one is unavailable in most cases.

Whatever is causing your issue, its not this.